build.sbt

```diff
@@ -26,6 +26,11 @@ commands := commands.value.filterNot { command =>
 
 ThisBuild / reproducibleBuildsCheckResolver := Resolver.ApacheMavenStagingRepo
 
+ThisBuild / pekkoCoreProject := true
```

Comment:
add a comment about this `pekkoCoreProject`?

Reply:
I mean it's kind of implied, this repo is the core pekko project?